cmd/vmonboarding.go

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package cmd
+
+import (
+	"errors"
+	"net"
+
+	"github.com/kubearmor/kubearmor-client/vm"
+	"github.com/spf13/cobra"
+)
+
+// vmPolicyAddCmd represents the command for vm onboarding
+var vmOnboardAddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "onboard new vm onto nonk8s control plane",
+	Long:  `onboard new vm onto nonk8s control plane`,
```

Comment:
```suggestion
	Short: "onboard new VM onto non-k8s control plane",
	Long:  `onboard new VM onto non-k8s control plane`,
```

Is this only for non-k8s control plane? I thought VM onboarding command remains same regardless of k8s or non-k8s .. right?

Reply:
It can be made common @nyrahul 
Currently the implementation is for non-k8s control plane alone. K8s control plane follows kubectl apply commands